scrunch/datasets.py

```diff
@@ -2537,6 +2537,7 @@ def replace_values(self, variables, filter=None, literal_subvar=False):
         if resp.status_code == 204:
             LOG.info('Dataset Updated')
             return
```

Comment:
Here I haven't changed the return value for `replace_values()` although I think we should always return the response instead of sometimes `None` and sometimes the response.
Making this change may break existing code that's relying on the response being None in some cases?